templates/etc/init.d/elasticsearch.RedHat.erb

```diff
@@ -31,7 +31,7 @@ if [ -f /etc/rc.d/init.d/functions ]; then
     . /etc/rc.d/init.d/functions
 fi
 
-exec="/usr/share/elasticsearch/bin/elasticsearch"
+exec="<%= homedir %>/bin/elasticsearch"
```

Comment:
These interpolations probably won't work unless a top-level `$homedir` exists in `manifests/service/init.pp` - it can just mirror `$elasticsearch::homedir`, but it needs to be brought into scope so the template can interploate it.